src/ILCompiler.WebAssembly/src/Compiler/WebAssemblyMetadataBlockingPolicy .cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Diagnostics;
```

Comment:
Is this part of a different change? (In case it matters for that change, you have a space in the file name before ```.cs```)

Reply:
removed the class as, yes, it was part of something else.